ledger/src/blockstore.rs

```diff
@@ -170,6 +174,50 @@ impl<T> AsRef<T> for WorkingEntry<T> {
     }
 }
 
+#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+pub struct LastFECSetCheckResults {
+    block_id: Option<Hash>,
+    is_retransmitter_signed: bool,
+}
+
+impl LastFECSetCheckResults {
+    pub fn report_metrics(&self, slot: Slot, bank_hash: Hash, cluster_type: ClusterType) {
```

Comment:
Do we need a separate function for `report_metrics`?
Can this be done right where the last FEC set is inspected? (or maybe a function which wraps the inspector function?)